packages/migrate-config/README.md

```diff
@@ -28,6 +28,13 @@ bunx @eslint/migrate-config .eslintrc.json
 
 The tool will automatically find your `.eslintignore` file in the same directory and migrate that into your new configuration file.
 
+If you receive any npm errors while running using `npx`, install the package manually first:
+
+```shell
+npm i @eslint/migrate-config
+npx @eslint-migrate-config .eslintrc.json
```

Comment:
typo?
```suggestion
npx eslint-migrate-config .eslintrc.json
```

Reply:
Actually, it should be `@eslint/migrate-config`

Reply:
Seems like both would work. `eslint-migrate-config` was defined: https://github.com/eslint/rewrite/blob/e0cfe185ec46fb4388663b6ff48d34f162859186/packages/migrate-config/package.json#L7

Reply:
Yes, both will work, I just think it's less confusing to stick to the same string everywhere.

Reply:
updated the suggestion. :)